config.example.storage.wikimedia.yaml

```diff
@@ -0,0 +1,113 @@
+# RESTBase wikimedia example config
+
+# Load some project templates. These are referenced / shared between domains
+# in the root_spec further down.
+default_project: &default_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - spec:
+                  paths:
+                    /parsoid_bucket:
+                      x-modules:
+                        - path: sys/multi_content_bucket.js
+                          options:
+                            grace_ttl: 1
+                            index_ttl: 864000
+                            renew_expiring: true
+                            delete_probability: 1
+                            table_name_prefix: parsoid_ng
+                            main_content_type:
+                              name: html
+                              value_type: blob
+                            dependent_content_types:
+                              - name: data-parsoid
+                                value_type: json
+                    /key_value:
+                      x-modules:
+                        - path: sys/key_value.js
+                    /key_rev_value:
+                      x-modules:
+                        - path: sys/key_rev_value.js
+          /{api:sys}:
+            x-modules:
+              - spec:
+                  paths:
+                    /table: &sys_table
+                      x-modules:
+                        - path: sys/table.js
+                          options:
+                            conf:
+                              backend: '{env(RB_TEST_BACKEND, sqlite)}'
+                              hosts: [localhost]
+                              keyspace: system
+                              username: cassandra
+                              password: cassandra
+                              defaultConsistency: one # or 'localQuorum' for production
+                              storage_groups:
+                                - name: default.group.local
+                                  domains: /./
+                              dbname: test.db.sqlite3 # ignored in cassandra, but useful in SQLite testing
+
+
+# The root of the spec tree. Domains tend to share specs by referencing them
+# using YAML references.
+spec_root: &spec_root
+  title: "The RESTBase Storage Service"
+  x-request-filters:
+    - path: lib/security_response_header_filter.js
+  paths:
+    /{domain}: *default_project
+    # A robots.txt to make sure that the content isn't indexed.
+    /robots.txt:
+      get:
+        x-request-handler:
+          - static:
+              return:
+                status: 200
+                headers:
+                  content-type: text/plain
+                body: |
+                  User-agent: *
+                  Allow: /*/v1/?doc
+                  Disallow: /
+
+# Finally, a standard service-runner config.
+info:
+  name: restbase
+
+services:
+  - name: restbase
```

Comment:
We probably want to chnge the name(s)

Reply:
to something we didn't decide on yet.

Reply:
Heh, we've settled on restbase, so no need to change :)